spec/services/doc_auth/lexis_nexis/responses/true_id_response_spec.rb

```diff
@@ -335,6 +338,12 @@
       expect(output).to include(:backtrace)
     end
 
+    it 'produces reasonable output for a bad DOB in the TrueID response' do
+      bad_response = described_class.new(failure_response_bad_dob, false, config)
```

Comment:
Instead of checking in an entire fixture file for 3 small fields, can we add a more precise unit test for the `parse_date` method with a few clearer arguments?

Reply:
I agree and I started with that, but `parse_date` is private, but I'll go ahead and test it explicitly. Done in 00c6789c7

Reply:
Yup! I see we used `.send` but we could also just make the method public! there are no rules in ruby, just vibes

Reply:
I'm going to leave it private - I wouldn't even test it except for the bug and how difficult it is to test it indirectly via the public interface.